src/etc/inc/util.inc

```diff
@@ -877,30 +877,6 @@ function subnet_size_by_netmask($iptype, $bits, $exact=false) {
 	}
 }
 
-
-function subnet_expand($subnet) {
-	if (is_subnetv4($subnet)) {
-		return subnetv4_expand($subnet);
-	} else if (is_subnetv6($subnet)) {
-		return subnetv6_expand($subnet);
```

Comment:
Yeah. I guess the function could be dynamically added, or something. But irrelevant really :)